Source/Core/DataExpression.cpp

```diff
@@ -255,28 +254,19 @@ class DataParser {
 		program.push_back(InstructionData{Instruction::NumArguments, Variant(int(num_arguments))});
 		program.push_back(InstructionData{instruction, Variant(std::move(name))});
 	}
-	void Variable(const String& expression) { VariableGetSet(expression, false); }
-	void Assign(const String& expression) { VariableGetSet(expression, true); }
+	void StaticVariable(const String& expression) { VariableGetSet(expression, false); }
+	void StaticAssign(const String& expression) { VariableGetSet(expression, true); }
```

Comment:
```suggestion
	void Variable(const String& data_address) { VariableGetSet(data_address, false); }
	void Assign(const String& data_address) { VariableGetSet(data_address, true); }
```
I think the term static has so many connotations in C/C++, that it might be confusing. I was thinking more about the parameter name. What do you think about this suggestion?